prusti-common/src/vir/ast/expr.rs

```diff
@@ -1414,6 +1414,7 @@ impl Expr {
                     | Expr::LetExpr(..)
                     | Expr::FuncApp(..)
                     | Expr::DomainFuncApp(..)
+                    | Expr::BackendFuncApp(_, _, _) // unsure what happens here and if this is correct
```

Comment:
It's better if you write questions like these into comments on your own PR and leave the code without them.